cranelift-codegen/shared/Cargo.toml

```diff
@@ -9,5 +9,10 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-packed_struct = "0.3"
-packed_struct_codegen = "0.3"
+packed_struct = { version =  "0.3", default-features = false }
+packed_struct_codegen = { version =  "0.3", default-features = false }
+
+[features]
+default = ["std"]
+std = ["packed_struct/std", "packed_struct_codegen/std"]
```

Comment:
Are those features actually used?

Reply:
Nope, I just thought it might be better to have the full feature set available in an std environment. I can get rid of it once I'm done with work today.